cis194/week7/tien/app/Main.hs

```diff
@@ -1,4 +1,18 @@
 module Main where
 
-main :: IO ()
-main = putStr "No use"
+import Editor
+import JoinList (JoinList(Empty, Single), (+++))
+import JoinListBuffer
+import Scrabble (scoreString)
+import Sized
+
+main =
+  runEditor editor $
+  foldr (+++) Empty $
```

Comment:
We can potentially use `fromString` here from Exercise 04

Reply:
How to apply instance methods? @sinogermany Thanks!

Reply:
> How to apply instance methods? @sinogermany Thanks!

OK this may not be the best way to do it but here you go: cis194/week7/daniel-deng/src/CIS194/Homework07/JoinListBufEditor.hs

@shineli1984 is there a way to remove the type annotation?